app/views/response_sets/_form.html.erb

```diff
@@ -46,11 +46,11 @@
     </div>
 
     <div class="hidden">
-      <% if defined?(parent_id) %>
-        <%= f.hidden_field :parent_id, :value => parent_id %>
-      <% end %>
       <%= f.hidden_field :version_independent_id %>
       <%= f.hidden_field :version %>
+      <% if defined?(response_set.parent) %>
```

Comment:
defined? is an odd method to use here. I would suggest response_set.parent.present? It is a minor not though.